### PR TITLE
Fix assembly and nuget descriptions

### PR DESCRIPTION
**Pull request by codebrain**

### CODE REVIEW

File: src/Elastic.Apm.NLog/Elastic.Apm.NLog.csproj [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.bat))\src\PublishArtifacts.build.props" />
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Title>Elastic APM NLog LayourRendrers</Title>
-    <Description>Inject APM TraceId and TransactionId automatically into your NLog logs</Description>
+    <Title>Elastic APM NLog Layout Renderers</Title>
+    <Description>Enrich NLog logs with APM TraceId and TransactionId.</Description>
```
Comment (russcam) [inline src/Elastic.Apm.NLog/Elastic.Apm.NLog.csproj]:
```suggestion
    <Description>Enrich NLog log messages with APM TraceId and TransactionId.</Description>
```

File: src/Elastic.Apm.SerilogEnricher/Elastic.Apm.SerilogEnricher.csproj [hunk 2]
```diff
@@ -1,14 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.bat))\src\PublishArtifacts.build.props"/>
-
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
-
     <Title>Elastic APM Serilog Enricher</Title>
-    <Description>Enrich your logs with APM TraceId and TransactionId automatically</Description>
-
+    <Description>Enrich Serilog logs with APM TraceId and TransactionId.</Description>
```
Comment (russcam) [inline src/Elastic.Apm.SerilogEnricher/Elastic.Apm.SerilogEnricher.csproj]:
```suggestion
    <Description>Enrich Serilog log messages with APM TraceId and TransactionId.</Description>
```

File: src/Elastic.CommonSchema.BenchmarkDotNetExporter/Elastic.CommonSchema.BenchmarkDotNetExporter.csproj [hunk 3]
```diff
@@ -3,9 +3,8 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-
-    <Title>ECS BencharkDotNet exporter</Title>
-    <Description>Exports BenchmarkDotNet benchmarks to ECS format</Description>
+    <Title>Elastic Common Schema (ECS) BencharkDotNet exporter</Title>
```
Comment (russcam) [inline src/Elastic.CommonSchema.BenchmarkDotNetExporter/Elastic.CommonSchema.BenchmarkDotNetExporter.csproj]:
```suggestion
    <Title>Elastic Common Schema (ECS) BenchmarkDotNet exporter</Title>
```

File: src/Elastic.CommonSchema.BenchmarkDotNetExporter/Elastic.CommonSchema.BenchmarkDotNetExporter.csproj [hunk 4]
```diff
@@ -3,9 +3,8 @@
...
+    <Description>Exports BenchmarkDotNet benchmarks to Elastic Common Schema (ECS) format</Description>
```
Comment (russcam) [inline src/Elastic.CommonSchema.BenchmarkDotNetExporter/Elastic.CommonSchema.BenchmarkDotNetExporter.csproj]: does it also index them into Elasticsearch, or just export them in that format?

File: src/Elastic.CommonSchema.Serilog/Elastic.CommonSchema.Serilog.csproj [hunk 5]
```diff
@@ -3,10 +3,8 @@
     <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.bat))\src\PublishArtifacts.build.props" />
     <PropertyGroup>
       <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
-      
       <Title>Elastic Common Schema (ECS) Serilog Formatter</Title>
-      <Description>Serilog TextFormatter that emits Elastic Common Schema (ECS) json, in a human scanable form.</Description>
-      
+      <Description>Serilog TextFormatter that emits Elastic Common Schema (ECS) JSON.</Description>
```
Comment (russcam) [inline src/Elastic.CommonSchema.Serilog/Elastic.CommonSchema.Serilog.csproj]:
```suggestion
      <Description>Serilog TextFormatter that formats log events in accordance with Elastic Common Schema (ECS).</Description>
```

File: src/Elastic.CommonSchema.BenchmarkDotNetExporter/Elastic.CommonSchema.BenchmarkDotNetExporter.csproj (hunk 4 above)
Comment (codebrain) [reply src/Elastic.CommonSchema.BenchmarkDotNetExporter/Elastic.CommonSchema.BenchmarkDotNetExporter.csproj]: Indexes too, via some helpers, will update

**Pull request merged**